pkgs/applications/misc/1password/default.nix

```diff
@@ -78,14 +78,10 @@ stdenv.mkDerivation {
 
   doInstallCheck = true;
 
-  installCheckPhase = ''
-    $out/bin/${mainProgram} --version
-  '';
+  versionCheckProgram = "${builtins.placeholder "out"}/bin/${mainProgram}";
+  versionCheckProgramArg = [ "--version" ];
```

Comment:
Aren't those the default?

Reply:
Default does help then version and it seemed to use pname instead of main program

Reply:
I can check again when I get home. But, I added the main program because it failed without it

Reply:
Yeah, it defaults to `pname` https://github.com/NixOS/nixpkgs/blob/ee8bdf4f09fdb4809ff8f2202decbfd02450edc3/pkgs/by-name/ve/versionCheckHook/hook.sh#L35 and the arg is to limit it to only checking `--version` instead of doing `--help`. https://github.com/NixOS/nixpkgs/blob/ee8bdf4f09fdb4809ff8f2202decbfd02450edc3/pkgs/by-name/ve/versionCheckHook/hook.sh#L45